factcast-factus-redis/src/main/java/org/factcast/factus/redis/tx/AbstractRedisProjection.java

```diff
@@ -26,18 +26,20 @@
 import org.factcast.factus.projection.Named;
 import org.factcast.factus.projection.WriterToken;
 import org.factcast.factus.projection.WriterTokenAware;
-import org.factcast.factus.projection.tx.AbstractOpenTransactionAwareProjection;
-import org.factcast.factus.redis.tx.RedisTransactional;
+import org.factcast.factus.redis.FactStreamPositionCodec;
+import org.factcast.factus.redis.RedisProjection;
+import org.factcast.factus.redis.RedisWriterToken;
 import org.redisson.api.*;
 
-abstract class AbstractRedisProjection extends AbstractOpenTransactionAwareProjection<RTransaction>
+@SuppressWarnings("java:S2142")
+abstract class AbstractRedisProjection
```

Comment:
I wonder if we should provide `AbstractRedisManagedProjection`/`AbstractRedisSubscribedProjection` similar to what we had in 0.7.x?